src/main/java/com/fivetran/external/com/amazonaws/services/kinesis/clientlibrary/lib/worker/Worker.java

```diff
@@ -526,6 +527,8 @@ void runProcessLoop() {
         } catch (AmazonDynamoDBException e) {
             throw e;
         } catch (Exception e) {
+            handleTrimmedDataAccessException(e);
```

Comment:
I guess you cant explicitly put this `TrimmedDataAccessException` with the `Exception` in the same block as L:529

Reply:
Yeah the problem is that the `TrimmedDataAccessException` is wrapped in another exception when it bubbles up here.